app/services/bulk_upload/lettings/year2024/row_parser.rb

```diff
@@ -467,6 +467,7 @@ def valid?
 
       fields.each do |field|
         next if errors.include?(field)
+        next if error.type == :skip_bu_error
```

Comment:
N.B. Copied this from sales since it makes sense not to turn this into a setup error for bulk upload.